frontend/src/component/releases/ReleasePlanTemplate/CreateReleasePlanTemplate.tsx

```diff
@@ -70,33 +88,69 @@ export const CreateReleasePlanTemplate = () => {
         }
     };
 
+    const onSidebarClose = () => {};
```

Comment:
Should we implement this one right away?

Reply:
It's a required prop on the sidebar, but not something I think we'll have use for? the use cases I can find all do some kind of navigation but there's not navigation to make here. could probably just set the prop to be `() => void` instead though